src/Libraries/Microsoft.Extensions.AI.Abstractions/CHANGELOG.md

```diff
@@ -1,5 +1,14 @@
 # Release History
 
+## 9.4.4-preview.1.25259.16
+
+- Added `AIJsonUtilities.TransformSchema` and supporting types.
+- Added `BinaryEmbedding` for bit embeddings.
```

Comment:
Is `BinaryEmbedding` exclusively for one-bit-per-dimension embeddings (i.e., ones where hamming distance is a sensible similarity metric), or does it mean "any binary blob that somehow represents an embedding"? Makes me wonder if the naming is too loose, or if the description here is too specific.

Reply:
It's intended for 1-bit embeddings. It's backed by BitArray.

Reply:
So would the name `BitEmbedding` be more applicable?

Reply:
We could of you think it's important, but binary embedding seems like the much more common term. When I search for bit embedding, the titles of the articles that come back are all about binary embeddings.

cc: @roji

Reply:
OK, that seems totally reasonable. I was focused on .NET conventions but agree that the favoring the industry-standard term for this makes sense.